core/src/main/scala/org/apache/spark/h2o/H2OConf.scala

```diff
@@ -27,7 +28,7 @@ import org.apache.spark.repl.h2o.H2OInterpreter
   * Configuration holder which is representing
   * properties passed from user to Sparkling Water.
   */
-class H2OConf(@transient val sc: SparkContext) extends Logging with InternalBackendConf {
+class H2OConf(@transient val sc: SparkContext) extends Logging with ExternalBackendConf with InternalBackendConf {
```

Comment:
yup, I would prefer 2nd idea because it's also previous API and it's easier to call H2OConf(sc) then H2OConf(sc.conf)